core/src/main/scala/kafka/message/Message.scala

```diff
@@ -19,7 +19,7 @@ package kafka.message
 
 import java.nio._
 
-import org.apache.kafka.common.record.TimestampType
+import org.apache.kafka.common.record.{Record, TimestampType}
```

Comment:
A meta clarification question: for all these classes, are we planning to get rid of them all at the same time when the old consumer is removed?